### ISSUE TITLE
Would you like to combine this project into the OSPAT?

### ISSUE THREAD

**Issue by voyagermedical** [kianenigma/awesome-psychedelics#3]
Hey,

Great project.

We have been working on something similar called the Open Source Psychedelic Assisted Therapy Project, it can be found here: https://github.com/unshakenme/OSPAT. We are actively looking for new members to contribute and I was wondering whether you would like to centralise our work so we can get economies of scale and make something with some momentum...?

**Comment by CheekeyMonkey** [comment]
Yes, great project. We need to centralise our expertise to get it off the ground. More info here: https://burnzero.com/Open_Source_Psychedelic-Assisted_Therapy

**Comment by kianenigma** [comment]
Sure, should we link to OSPAT here? 

Honestly, I am not maintaining this repo too much myself. My hope is that more people would add links through PRs, which you are welcome to do for OSPAT as well :)

**Comment by CheekeyMonkey** [comment]
Hey, yes, would be great to keep this repository open but have everything point towards [](https://github.com/unshakenme/OSPAT). If you add yourself as a participant to the OSPAT repo and the make a fork  using the files you have collected here. we will merge it into the main.

**Comment by EggzyGallahad** [comment]
Hi, I'm interested in joining forces for collaboration. Please check the following:

https://www.psychedelicsdaily.com/social-contract/

https://www.psychedelicsdaily.com/open-psychedelics/

**Comment by VoytekBereza** [comment]
The best way to collaborate is to become active in the community i.e. through helping with issues. Another way is to fork the repository add some information and then apply for it to be merged (we have accepted numerous revisions in the past).

**Comment by tim-tepia** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1500 in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @msk-santo, @akhilmantha, @itsjameszhao, @Astrip21, @winar-jin, @enochkan, @upmorpheus, @tigerneil, @meierna, @guim4dev, @ezingme, @pgkgone, @naijab, @kdev, @Bioboy-Learning, @paveda, @emekanw, @saadjhk, @ajott1, @Victor4X, @shenjiayu, @dulumao, @imnolte, @ihao, @dojinkimm, @bingodd, @liqianjie, @let5sne, @geordanex, @KhantZinThu-Dev, @edac713, @CoolSpot, @LoboMix11, @quiz42, @EItanya, @Gxine07, @SwaroopH, @igdeman, @davidkennedydev, @michaelkirschbaum, @vikrant6661, @DMuld, @zhanhai, @balupton, @antonicarol, @anhcraft, @patrickribbsaeter, @CullenWallace, @nagasssi, @wardtwits
